networking/multiple_networks/primary_networks/about-user-defined-networks.adoc

```diff
@@ -18,6 +18,8 @@ image::527-OpenShift-UDN-isolation-012025.png[The namespace isolation concept in
 [NOTE]
 ====
 Support for the Localnet topology on both primary and secondary networks will be added in a future version of {product-title}.
```

Comment:
Confirmed with @maiqueb, localnet is supported only on secondary nets.

Reply:
@weliang1 Was it confirmed that localnet will always be for secondary network and never for primary network even in future?

Reply:
Note about cgroup lgtm

Reply:
there's no use case for localnet on primary networks.

I can't promise it never will, but I really don't think it makes sense to have it as primary. 

... but I'm just a developer.

Maybe we need to raise this to product owners ?

Reply:
Fixed over here https://github.com/openshift/openshift-docs/pull/88986/files#diff-3b931ff4d98ce8b85a1482e4b33a95a0ca205fb5868b95611ec0a0073ae7584dL18-L22

Reply:
Make this note as Support for the Localnet topology only on secondary network.

Reply:
This is outside the scope of this PR.